examples/hugo-quickstart/resources/_gen/assets/css/ananke/css/main.css_bb5467e0521bbea6b1e66429f6ec028e.json

```diff
@@ -0,0 +1 @@
+{"Target":"ananke/css/main.min.css","MediaType":"text/css","Data":{}}
```

Comment:
@jeffsee55  can these `resources/_gen/**`  files be git ignored? or are they needed?

Reply:
I'm not actually sure, if it gets annoying we can always add the `gitignore`